website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx

```diff
@@ -1,14 +1,18 @@
 ### Vault is storing references to ephemeral sub-loggers leading to a memory leak
 
-Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
-a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
-There is no workaround.
-
 #### Affected versions
 
-This issue affects Vault Community and Enterprise versions:
-- 1.13.7+
-- 1.14.3+
-- 1.15.0+
+This memory leak affects Vault Community and Enterprise versions:
+
+- 1.13.7 - 1.13.9
+- 1.14.3 - 1.14.5
+- 1.15.0 - 1.15.1
 
-A fix will be issued in the next release
+This change that introduced this leak will be reverted in the next release
+
+#### Issue
+Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
```

Comment:
I remember a version of this update that had more information about what is a sublogger and what the impact of the subloggers issue revert was - was that update in this PR or a different PR? I don't think this description covers the expected behavior now that we've reverted the change and the memory leak is going away.

Reply:
We split that off into its own known issue. I'll update this to refer to that issue. There was concern over too much detail overwhelming the reader in the previous version since it was two issues in one.